tests/Resources/ti.ui.imageview.test.js

```diff
@@ -240,11 +240,10 @@
 		});
 
 		it('fires error event for URL pointing at resource that does not exist', function (finish) {
-			if (OS_IOS) {
-				this.timeout(21000); // default timeout of underlying request is 20 seconds, so let's wait one extra
-			}
-			win = Ti.UI.createWindow();
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.imageview.test.js;
  Line: 7;
-->

- :warning: tests/Resources/ti.ui.imageview.test.js line 7 – 'OS_IOS' is defined but never used. (no-unused-vars)